helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -428,4 +432,20 @@ public String toString() {
     return verifierName + "(" + _clusterName + "@" + _zkClient + "@resources["
        + (_resources != null ? Arrays.toString(_resources.toArray()) : "") + "])";
   }
+
+  private class ReadOnlyWagedRebalancer extends org.apache.helix.controller.rebalancer.waged.ReadOnlyWagedRebalancer {
```

Comment:
Let's have a different name for this private class. I would still prefer the old name DryrunWagedRebalancer.
But feel free to have a better name for this method.

Reply:
Okay. Changed to Dryrun